src/udf/utils.c

```
@@ -92,7 +98,10 @@ JSContext *UDF_GetExecutionJSContext
 	JSContext *js_ctx = JS_NewContext(js_rt) ;
 	ASSERT (js_ctx != NULL) ;
 
-	UDF_CTX_RegisterClasses (js_ctx) ;
+	UDF_RegisterGraphObject  (js_ctx) ;
```

Cppcheck (2.19.0)

[error] Common realloc mistake

(memleakOnRealloc)